src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssPropertiesDotComa = sourceString
+    .split(';')
+    .map((item) => item.trim())
+    .filter((item) => item.length > 0);
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. Methods like `split`, `map`, and `filter` don't mutate the original array, so intermediate variables are not needed unless they improve readability.